### DISCUSSION TITLE
Is that possible for integrated with mlange-42/git-graph?

### DISCUSSION THREAD

**Discussion by sincebyte** [magit/magit#4656] [Ideas]
cause Magit default point line diagram is not intuitive enough. So is that possible for integrated with mlange-42/git-graph?
The url is https://github.com/mlange-42/git-graph.
![截屏2022-04-09 13 42 40](https://user-images.githubusercontent.com/7248023/162558248-6576b992-5da2-44a1-80b0-1b8fd76128b5.png)

**Comment by tarsius** [comment]
I plan to eventually show log graphs along these lines, see #2989. Whether that is going to happen soon and whether that will involve some third-party tool I don't know.

**Reply by tarsius** [reply]
I've added a note about this tool to #2989.
